src/process_csv_evaluations.py

```
@@ -0,0 +1,288 @@
+#!/usr/bin/env python3
+"""
+Script to process CSV data and run OCR evaluations using Gemini OCR.
+Processes each row in the CSV, calls extract_text function, and saves results.
+"""
+
+import csv
+import json
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)